io/src/main/scala/com/velocidi/apso/io/SftpFileDescriptor.scala

```diff
@@ -18,33 +18,22 @@ import net.schmizz.sshj.xfer.InMemorySourceFile
 
 import com.velocidi.apso.Logging
 
-/** A `FileDescriptor` for files served over SFTP. This file descriptor only supports absolute paths.
-  * The SSH connections for a given host are pooled.
+/** A `FileDescriptor` for files served over SFTP. This file descriptor only supports absolute paths. The SSH
+  * connections for a given host are pooled.
   *
   * The URI for this `FileDescriptor` should be in the format:
-  * - `sftp://<username>@<hostname>:<port>/<absolute-path>`
+  *   - `sftp://<username>@<hostname>:<port>/<absolute-path>`
   *
-  * Both the username and port are optional. Additionally, the credentials config expects an object
-  * with the following format:
+  * Both the username and port are optional. Additionally, the credentials config expects an object with the following
+  * format:
   *
-  * `sftp {
-  *    default = {
-  *      username = <username>
-  *      password = <password>
-  *    }
-  *  }`
+  * `sftp { default = { username = <username> password = <password> } }`
```

Comment:
Can we replace this by source code blocks to avoid reformat?

```suggestion
  * {{{
  * sftp {
  *   default = {
  *     username = <username>
  *     password = <password>
  *   }
  * }
  * }}}
```